api/all.yaml

```diff
@@ -2278,6 +2375,138 @@ components:
       required:
         - principal
         - privileges
+    Commit:
+      type: object
+      properties:
+        table_id:
+          type: string
+          description: The ID of the table to commit to. In the context of UC, this is the UUID that uniquely identifies a table.
+        table_uri:
+          type: string
+          description: |
+            The URI of the storage location of the table. If the table_id exists but the table_uri is 
+            different from the one previously registered (e.g., if the client moved the table), the request will fail.
+        commit_info:
+          $ref: '#/components/schemas/CommitInfo'
+        latest_backfilled_version:
+          type: integer
+          format: int64
+          description: |
+            The highest version of the commits that have been backfilled for this table; meaning UC no longer 
+            needs to keep track of commits of versions <= this version. At least one of commit_info and 
+            latest_backfilled_version must be present.
+        metadata:
+          $ref: '#/components/schemas/Metadata'
+        protocol:
+          $ref: '#/components/schemas/Protocol'
+      required:
+        - table_id
+        - table_uri
+    Metadata:
```

Comment:
Add `name`  to Metadata

Reply:
Based on previous discussion I had removed name and description, do we need just name or both?

Reply:
Apologies for the back and forth. Need both `name` and `description`.